samples/snippets/src/main/java/com/example/analyticsadmin/QuickstartSample.java

```diff
@@ -0,0 +1,80 @@
+ * limitations under the License.
+ */
+package com.example.analytics;
+
+/* This application demonstrates the usage of the Analytics Admin API using
+ service account credentials. For more information on service accounts, see
+ https://cloud.google.com/iam/docs/understanding-service-accounts
+
+ The following document provides instructions on setting service account
+ credentials for your application:
+ https://cloud.google.com/docs/authentication/production
+
+ In a nutshell, you need to:
+
+ 1. Create a service account and download the key JSON file.
+ https://cloud.google.com/docs/authentication/production#creating_a_service_account
+
+ 2. Provide service account credentials using one of the following options:
+ - set the GOOGLE_APPLICATION_CREDENTIALS environment variable, the API
+ client will use the value of this variable to find the service account key
+ JSON file.
+
+ https://cloud.google.com/docs/authentication/production#setting_the_environment_variable
+
+ OR
+
+ - manually pass the path to the service account key JSON file to the API client
+ by specifying the keyFilename parameter in the constructor.
+ https://cloud.google.com/docs/authentication/production#passing_the_path_to_the_service_account_key_in_code
+
+ To run this sample using Maven:
+   cd java-analytics-admin/samples/snippets
+   mvn exec:java -Dexec.mainClass="com.example.analytics.QuickstartSample"
+ */
+
+// [START analytics-admin_quickstart]
+
+import com.google.analytics.admin.v1alpha.Account;
+import com.google.analytics.admin.v1alpha.AnalyticsAdminServiceClient;
+import com.google.analytics.admin.v1alpha.AnalyticsAdminServiceClient.ListAccountsPage;
+import com.google.analytics.admin.v1alpha.AnalyticsAdminServiceClient.ListAccountsPagedResponse;
+import com.google.analytics.admin.v1alpha.ListAccountsRequest;
+
+public class QuickstartSample {
+
+  public static void main(String... args) throws Exception {
+    // Instantiates a client using default credentials.
+    try (AnalyticsAdminServiceClient analyticsAdmin = AnalyticsAdminServiceClient.create()) {
+      // Calls listAccounts() method of the Google Analytics Admin API and prints
+      // the response for each account.
+      ListAccountsPagedResponse response = analyticsAdmin
+          .listAccounts(ListAccountsRequest.newBuilder().build());
+      for (ListAccountsPage page : response.iteratePages()) {
+        for (Account account : page.iterateAll()) {
+          System.out.printf("Account name: %s\n", account.getName());
+          System.out.printf("Display name: %s\n", account.getDisplayName());
+          System.out.printf("Country code: %s\n", account.getCountryCode());
+          System.out.printf("Create time: %s\n", account.getCreateTime().getSeconds());
+          System.out.printf("Update time: %s\n", account.getUpdateTime().getSeconds());
+          System.out.println();
+        }
+      }
+    }
+  }
+}
+
+// [END analytics-admin_quickstart]
```

Comment:
This tag was named this way in order to stay consistent with autogenerated tags in pom.xml (now removed since BOM is not used). Will update per suggestion for readability.